errors.ts

```diff
@@ -66,7 +67,7 @@ export function installUncaughtExceptionListener(): void {
 		 	var analyticsService = $injector.resolve("analyticsService");
```

Comment:
Ok, it's kind of fun here. In case you pass incorrect -- option (for example $ appbuilder --gosho) we fail when bootstrap is requiring options.ts from appbuilder-cli. As the "require" is not finished, when we resolve analyticsService here (which has "import options = require("../options");" on line 7, we try to parse options again and we throw error once again. So the result is that our error for incorrect option is catched, after that we try resolving analyticsService we throw it again and we catch it two lines below.
The output is:
Error: The option 'gosho' is not supported. To see command's options, use '$ appbuilder emulate --help'. To see all commands use '$ appbuilder help'.
    at Errors.fail (D:\Work\icenium-cli\lib\common\errors.ts:98:68)
    at Errors.failWithoutHelp (D:\Work\icenium-cli\lib\common\errors.ts:107:7)
    at <anonymous> (D:\Work\icenium-cli\lib\common\helpers.ts:88:11)
    at Function.forEach (D:\Work\icenium-cli\node_modules\lodash\dist\lodash.js:3297:15)
    at validateYargsArguments (D:\Work\icenium-cli\lib\common\helpers.ts:84:4)
    at Object.getParsedOptions (D:\Work\icenium-cli\lib\common\helpers.ts:77:1)
    at Object.<anonymous> (D:\Work\icenium-cli\lib\options.ts:48:21)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
Error while reporting exception: Exception: The option 'gosho' is not supported. To see command's options, use '$ appbuilder emulate --help'. To see all commands use '$ appbuilder help'.